### DISCUSSION TITLE
Tipo Richiedente per il caso d'uso Trascr_012, nel caso di Sentenza da Tribunale.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#1065] [Miglioramenti]
Buongiorno,

Per il caso d'uso Trascr_012 ('Trascrizione del decreto prefettizio di cambiamento del nome e cognome'), sempre su nostra richiesta, la denominazione dell'allegato obbligatorio, è stata modificata da 'Decreto Prefettizio', a 'Decreto Prefettizio / Sentenza', permettendo così di gestire, con questo caso d'uso, anche l'evento nel caso di Sentenza da Tribunale.

Nel caso di Sentenza da Tribunale, però, non c'è un Soggetto richiedente, in quanto arriva direttamente comunicazione all'USC del Comune, che provvede all'adempimento conseguente.
Per procedere con la validazione, in questo caso, stiamo valorizzando il tipoRichiedente = 3 ('Soggetto' da ANSC_64), ma sarebbe forse più corretto identificare il caso specifico, aggiungendo alle tipologie di richiedente, il 'Tribunale', senza la richiesta di un Soggetto richiedente, per il caso d'uso.

Chiediamo quindi, se possibile aggiungere una nuova tipologia di richiedente (id:5, descrizione:Tribunale), alla codifica ANSC_64, con la gestione indicata, per il caso d'uso Trascr_012.

In attesa di riscontro,

cordiali saluti.

**Comment by asqueglia** [comment]
Buon pomeriggio, nel caso di trascrizione del decreto prefettizio di cambio nome e cognome richiesto da tribunale è possibile far riferimento al caso d'uso [1.3.3.2] Trascrizione del decreto prefettizio o sentenza di cambiamento del nome e cognome su richiesta dell'autorita' pubblica o consolare  specificando come autorità mittente il tribunale stato italia e comune italiano. Chiudiamo pertanto la discussion
